File: LoginDagger/app/src/main/java/com/charlye934/loginddagger/MainActivity.java
package com.charlye934.loginddagger;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText firstName, lastName;
    Button loginButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        firstName = findViewById(R.id.etFirstName);
        lastName = findViewById(R.id.etLastName);
        loginButton = findViewById(R.id.btnLogin);

        loginButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(), "Prueba",Toast.LENGTH_SHORT).show();
            }
        });

    }
}

File: JetPack/NavigationComponent/app/src/main/java/com/charlye934/navigationcomponent/MainActivity.kt
package com.charlye934.navigationcomponent

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.charlye934.navigationcomponent.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {

    private lateinit var bindign: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        bindign = DataBindingUtil.setContentView(this, R.layout.activity_main)
    }
}